docs/node_resource_handling.md

```diff
@@ -0,0 +1,87 @@
+## Node Resource Handling In Kuberenetes
+
+An aspect of Kubernetes clusters that is often overlooked is the resources non-pod components require to run, such as:
+
+* Operating system components i.e. `sshd`, `udev` etc.
+* Kubernetes system components i.e. `kubelet`, `container runtime` (e.g. Docker), `node problem detector`, `journald` etc.
+
+As you manage your cluster, it's important that you are cognisant of these components because if your critical non-pod components don't have enough resources, you might end up with a very unstable cluster.
+
+### Understanding Node Resources
+
+Each node in a cluster has resources available to it and pods scheduled to run on the node may or may not have resource requests or limits set on them. Kubernetes schedules pods on nodes that have resources that satisfy the pod's specified requirements. As a result, pods are sort of [bin-packed][4] onto the nodes in a best effort attempt to utilize as much of the resources available with as few nodes as possible.
```

Comment:
@justinsb About your feedback ...

> you probably want to say something like `...(although for a reliable cluster you really should set limits)`

I see you've seen that I addressed that in [the list at the bottom][1]. 😄 

> inserting line-breaks makes it easier, especially for github commenting :-)

Addressed in https://github.com/kubernetes/kops/pull/2992/commits/14359657c0eb39b8b108977604a8af45e142f0ae.

> Maybe instead of the fuzzy ...

Addressed in https://github.com/kubernetes/kops/pull/2992/commits/1bd329ad93c5682ca564dd0b741440a852b122fd.

[1]: https://github.com/kubernetes/kops/pull/2992#discussion_r131305491